packaging/docker/entrypoint.py

```diff
@@ -2,17 +2,54 @@
 import json
 import logging
 import os
+import shutil
 import subprocess
 import time
+from datetime import datetime, timedelta
+from glob import glob
 from pathlib import Path
-from typing import Dict, Optional, Any, List
+from typing import Any, Dict, List, Optional
 
 logger = logging.getLogger('monitor')
 logging.basicConfig(level=logging.DEBUG)
 
 DEFAULT_LOG_LEVEL = 'critical'
 
 
+def can_delete(file: str, cutoff: int) -> bool:
+    return int(os.stat(file).st_mtime) <= cutoff or Path(file).name.startswith('_MEI')
+
+
+def cleanup_tmp() -> None:
+    logger.info('Preparing to cleanup tmp directory')
+    tmp_dir = glob('/tmp/*')
+    cache_cutoff = datetime.today() - timedelta(hours=6)
+    cutoff_epoch = int(cache_cutoff.strftime("%s"))
+    to_delete = filter(lambda x: can_delete(x, cutoff_epoch), tmp_dir)
+
+    deleted = 0
+    skipped = 0
+
+    for item in to_delete:
+        path = Path(item)
+        if path.is_file():
+            try:
+                path.unlink()
+                deleted += 1
+            except PermissionError:
+                skipped += 1
+                continue
+        else:
+            try:
+                shutil.rmtree(item)
+                deleted += 1
+            except OSError:
+                skipped += 1
+                continue
```

Comment:
nitpick to not have both levels of indentation

```suggestion
                deleted += 1
                continue
            except PermissionError:
                skipped += 1
                continue
        # else
        try:
            shutil.rmtree(item)
            deleted += 1
        except OSError:
            skipped += 1
```

Also the last continue is not needed at the last except.